### ISSUE TITLE
Conflicting version of Microsoft.Owin

### ISSUE THREAD

**Issue by APErebus** [marcinbudny/applicationinsights-owinextensions#37]
I just installed the latest 0.8.0 version of this package and am getting `Microsoft.Owin` package version issues.

According to the package details, `Microsoft.Owin >= 2.0.2` is supported, however when I add it, I get this issue (we are currently running 3.1.0 for a lot of bad reasons, but I can't upgrade easily)

![image](https://user-images.githubusercontent.com/332730/86717215-4f20c900-c065-11ea-8331-9aadd3f3aa57.png)

A binding redirect like 
```
			<dependentAssembly>
				<assemblyIdentity name="Microsoft.Owin" publicKeyToken="31BF3856AD364E35" culture="neutral"/>
				<bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="3.1.0.0"/>
			</dependentAssembly>
```
works, but I don't think it should be necessary?

Is the package being built against the incorrect `Microsoft.Owin` version (compared to the listed dependencies)?

**Comment by marcinbudny** [comment]
It seem that in fact version 0.8.0 was built against incorrect Owin version. Thank you for spotting this. I will provide a fix soon.

**Comment by marcinbudny** [comment]
Resolved in 0.8.1
